Bump to SVGO 2.x

aaronshekey opened the pull request:
This PR gets us on the latest and greatest from SVGO. The output is subtly different, and the config API has changed, but everything looks the same, visually.

aaronshekey committed:
Bump Webpack to 5.30.0

aaronshekey committed:
Bump SVGO to 2.3.0 and fix config

aaronshekey committed:
Run the build

abovedave committed:
no need to force it

abovedave committed:
use extendDefaultPlugins

abovedave committed:
package-lock

abovedave committed:
build

abovedave reviewed (approved):
Looks like the config api had changed a bit, I gave it a tidy and used `extendDefaultPlugins` utility they suggest. Also found a new setting for `noSpaceAfterFlags` on a few options.

aaronshekey committed:
Namespace gradients

Now that cleanupIDs is enabled, I needed to amend this regex. I wasn't able to get prefixIds to work at all, but should accomplish this same thing.
